### DISCUSSION TITLE
AlienFX-gui works on dell g15 5515?

### DISCUSSION THREAD

**Discussion by williamcastanho** [T-Troll/alienfx-tools#231] [Q&A]
Hi, guys. At first, I want to congratulate, and thanks all the team for this software. I'm using the fan control and works perfect. Does what the Dell work team can't do correctly.
But I can't make the alienfx-gui works properly. I did read the wiki page and follow the tutorial, but even doing the steps, the lights do not turn on.
I tried to use the alienfx-cli on powershell, so did work.
Can anyone help me?

**Comment by T-Troll** [comment]
Thanks.
The team is only me, in fact, and a lot of guys around the world who help me with different hardware testing. 
See Discord server from readme, most of the testing and operational discussions there.

G15 can have 2 modifications - with Alienware keyboard and without. First one is supported as a common APIv4 chip (4 keyboard zones), second one doesn't have light control.
All you need to do (in first case) is to find which lightID use for it - and don't apply other models, it's different from model to model.
So open GUI, switch to "Devices" tab and define lights manually as described in Wiki (setting it to grid as well!). It's a one-time operation.
